### PR TITLE
Feature/csv creator

### PR DESCRIPTION
**Pull request by dgenelov**
Include methods for creating a .csv-file

### CODE REVIEW

**Comment by sarakinell** [comment]
Good job! Annie and I have gone through the code and have feedback and some issues we'd like to see solved before the code is merged with develop.
- [x] All methods for writing/reading .csv-files should be in a separate class (e.g CsvHandler)
- [x] StorageHandler does not follow the design by contract nor the input/output agreement stated in Trello. 
- [x] The functionality of StorageHandler overlaps with other methods in MainActivity, e.g. creating a reference to storage.
- [x] There is no reference or usage of StorageHandler in MainActivity or elsewhere at the moment.
- [ ] getUrlAsync needs to be commented (@JonathanFager?)

It would be great if these things could be adjusted before we merge. The code is quite messy at the moment which makes it hard to follow.

**Comment by dgenelov** [comment]
Thanks girls, great feedback!
The StorageHandler class was not supposed to be included in this pull request. I deleted the StorageHandler.java file - it is still accessible on the feature/upload branch. 

getUrlAsync method needs to be commented but was not a part of this pull request. Perhaps @JonathanFager can do this in another branch?

We first tried making the .csv-file methods in another class, but got a lot of error messages. My initial thought was to make this as a part of the later refactoring, as we will need to refactor our whole MainActivity.java class, and we might as well do this in a new branch. But I see your point! I will have a go at making it work in a new class and then commit my changes.

**Comment by dgenelov** [comment]
I made it! Now there's a new class, CsvHandler, like you asked for :). Please revisit the changes and merge or give further comments if necessary.

**Comment by sarakinell** [comment]
Perfect, @dgenelov! Very well done with the javadoc commenting and the testing. Great example method that should be documented as unit test!

Sounds good, getUrlAsync needs to be fixed now, maybe a hot fix would do the job.

Great that StorageHandler was removed. I noted your comment to "... creating a java file object if Fager's uploadFileToFirebase-method will need this ..." and would recommend you to try and keep a closer dialogue with @JonathanFager. As it was yesterday, both the Uri and the File is created in Jonathan's StorageHandler.

Glad to hear you figured out how to create the CsvHandler class! Looks flawless!
